legacy-documents/aaelf32/ihi0044_H/IHI0044G_aaelf.html

```diff
@@ -0,0 +1,5669 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>

```

Comment:
This file has the wrong name! It's in the subdirectory for version `H` of the document, but its name has `G` in it.

The version number inside the file is `H`, so I think it's only the file name that's wrong.

Reply:
Thanks, done, and renamed the PDF as well.

Reply:
also changed `.html` to `.xhtml`